releases/v2.10.0/bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -892,5 +892,5 @@ spec:
   maturity: beta
   provider:
     name: MongoDB, Inc
-  version: 0.0.0
+  version: 2.10.0
```

Comment:
Thanks to your very sharp eye @helderjs we found these to be wrong in the tag itself!

I have fixed it from `main`